admin/js/editor.js

```diff
@@ -2,13 +2,6 @@ const {unregisterBlockStyle, registerBlockStyle} = wp.blocks;
 const {__} = wp.i18n;
 
 wp.domReady(() => {
-  // Remove Take Action and Campaign covers styles for Covers block in campaigns
```

Comment:
This piece of code was not moved to the master theme.
Is it no longer necessary?

Reply:
No, I've discussed it with @comzeradd and since we want to retire Campaign pages it felt unnecessary